sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -106,6 +106,12 @@ class ReturnHelper {
 
 } // anonymous namespace
 
+// Get Next Power-of-2 larger than k for given n
+static inline constexpr unsigned int getNextPowerOf2(unsigned int n,
```

Comment:
I see sycl/include/sycl/ext/intel/experimental/esimd/detail/util.hpp
Already contains isPowerOf2() and getNextPowerOf2().